src/components/graph/NodeTooltip.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div
+    v-if="tooltipText"
+    ref="tooltipRef"
+    class="node-tooltip"
+    :style="{ left, top }"
+  >
+    {{ tooltipText }}
+  </div>
+</template>
+
+<script setup lang="ts">
+import { nextTick, ref, onBeforeUnmount, watch } from 'vue'
+import { LiteGraph } from '@comfyorg/litegraph'
+import { app as comfyApp } from '@/scripts/app'
+import { useNodeDefStore } from '@/stores/nodeDefStore'
+import { useSettingStore } from '@/stores/settingStore'
+
+let idleTimeout: NodeJS.Timeout
```

Comment:
I think this should just be `number` as we are in the browser environment.

Reply:
![image](https://github.com/user-attachments/assets/1a513f5e-786c-42a2-8939-27eab9a44a27)
I would then need to cast it

Reply:
`ReturnType<typeof setTimeout>` is probably the other option, but that will still be NodeJS.Timeout

Reply:
You can explicitly call `window.setTimeout` here.